packages/pds/src/app-migrations/remove-scenes.ts

```diff
@@ -0,0 +1,66 @@
+import assert from 'assert'
+import AppContext from '../context'
+import Database from '../db'
+import { appMigration } from '../db/leader'
+
+const MIGRATION_NAME = '2023-03-23-remove-scenes'
+const SHORT_NAME = 'remove-scenes'
+
+export async function removeScenesMigration(ctx: AppContext) {
+  await appMigration(ctx.db, MIGRATION_NAME, (tx) => main(ctx, tx))
+}
+
+async function main(ctx: AppContext, tx: Database) {
+  console.log(SHORT_NAME, 'beginning')
+  tx.assertTransaction()
+
+  const res = await tx.db
+    .selectFrom('did_handle')
+    .where('actorType', '=', 'app.bsky.system.actorScene')
+    .select('did')
+    .execute()
+
+  const sceneDids = res.map((row) => row.did)
+
+  if (sceneDids.length < 1) {
+    console.log('no scenes to delete')
+    return
+  }
+
+  await Promise.all(
+    sceneDids.map(async (did) => {
+      await ctx.services.record(tx).deleteForUser(did)
+      await ctx.services.repo(tx).deleteRepo(did)
+      await ctx.services.account(tx).deleteUser(did)
+    }),
+  )
+
+  console.log(`deleted ${sceneDids.length} scenes`)
+
+  // sanity check
+
+  const check1 = tx.db
+    .selectFrom('ipld_block')
+    .where('creator', 'in', sceneDids)
+    .selectAll()
+    .execute()
+
+  const check2 = tx.db
+    .selectFrom('did_handle')
+    .where('did', 'in', sceneDids)
+    .selectAll()
+    .execute()
+
+  const check3 = tx.db
+    .selectFrom('record')
+    .where('did', 'in', sceneDids)
+    .selectAll()
+    .execute()
```

Comment:
I think this will also remove some assertions, if we want to do any dummy-checking in the app view.

Reply:
Nice I added that check 👌